### PR TITLE
fix(people): remove superfluous loadPeopleFromUrl

### PR DESCRIPTION
**Pull request by hazzadous**
Previously I'd defined the `loadPeopleFromUrl` action on which I'd added
typing. This action is already defined as a loader. This loader didn't
have the arg types specified so i've added the type from the standalone
action to the loader.

Note that we're also listening on `loadPeopleFromUrl`, the implicit payload 
for the action will be the parameters of the loader action.

Resolves https://github.com/PostHog/posthog/pull/7099#discussion_r749965851

### CODE REVIEW

**Comment by hazzadous** [comment]
Just trying to fix the e2e failure, although I think it's unrelated

**Comment by hazzadous** [comment]
> Just trying to fix the e2e failure, although I think it's unrelated

Looks related https://posthog.slack.com/archives/C0113360FFV/p1637050660342800

**Comment by hazzadous** [comment]
Have merged master, although still failing. I'll do some further debugging locally.

**Comment by hazzadous** [comment]
@mariusandra I just merged in master again and looks like it's all passing now 🎉  i.e. ready for review if you have the time 🙏

**Review by mariusandra — APPROVED** [review]
Hey, I guess this is in response to [this comment](https://github.com/PostHog/posthog/pull/7099#discussion_r749965851)?
Either of these options, the before and after, works, and is technically equivalent, so feel free to merge as you'd like. 

Though, I personally like the explicitness of defining actions in `actions`, especially if they're used elsewhere as well (like in `reducers` in this case). But it really doesn't matter.

The second remark I had that in this list of similar things:

```ts
  setCohortModalVisible: (visible: boolean) => ({ visible }),
  loadPeople: (peopleParams: PersonModalParams) => ({ peopleParams }),
  loadPeopleFromUrl: (props: LoadPeopleFromUrlProps) => props,
```

this line

```
  loadPeopleFromUrl: (props: LoadPeopleFromUrlProps) => props,
```

could be more like this:

```
  loadPeopleFromUrl: (urlProps: LoadPeopleFromUrlProps) => ({ urlProps }),
```

However, I didn't look too deeply before I made my comment (sorry), and seeing how `LoadPeopleFromUrlProps` is used, and how nice it is to directly destructure the props directly in the loader's arguments:

```ts
            loadPeopleFromUrl: async ({
                url,
                funnelStep,
                breakdown_value = '',
                date_from = '',
                action,
                label,
                pathsDropoff,
            }) => {
```

I think it's valid to ignore this suggestion as well. It really doesn't matter either.

So, in summary, any way is fine. Merge if you want to, skip if not. Sorry if my comment made you lose time here...

**Comment by hazzadous** [comment]
> Hey, I guess this is in response to this comment?

Yup. Looking at it closer it looks like `loadPeople` is both explicitly a loader and an action, so perhaps best to keep consistency and not merge.

> So, in summary, any way is fine. Merge if you want to, skip if not. Sorry if my comment made you lose time here...

We've got to run out the clock somehow, right? 🙏
